src/context/videoContext.js

```diff
@@ -1,12 +1,113 @@
+import axios from "axios";
 import { createContext, useContext, useReducer } from "react"
+import { toast } from "react-toastify";
+import { initialVideoState, videoReducer } from "reducer/videoReducer";
 
 const VideoContext = createContext();
 
 function VideoProvider({ children }) {
-    const [videoState, videoDispatch] = useReducer();
-    <VideoContext.Provider value={{ videoState, videoDispatch }}>
-        {children}
-    </VideoContext.Provider>
+
+    const [videoState, videoDispatch] = useReducer(videoReducer, initialVideoState);
+
+    const encodedToken = localStorage.getItem("token");
+
+    const likeHandler = async (video) => {
+        try {
+            const response = await axios.post(
+                "/api/user/likes",
+                { video: video },
```

Comment:
suggestion - you can replace this with `{video}` and it would work. According to the ES6 syntax, it will take the `key` as the variable name.